www/config.xml

```diff
@@ -48,9 +48,10 @@
 
     <!-- Plugins -->
 
+	<gap:plugin name="org.apache.cordova.camera" />
```

Comment:
can you style this so that it conforms with how we link other plugins into the app?